main.go

```diff
@@ -618,12 +624,6 @@ func inspect(diff gitDiff) []Leak {
 		skipLine bool
 	)
 
-	for _, re := range whiteListFiles {
```

Comment:
We need to add this to https://github.com/zricethezav/gitleaks/blob/master/github.go#L53-L57. Test for github PR is failing with these changes